### PR TITLE
Adjust notifications and profile modal on mobile on default theme

### LINKED ISSUE #23099: [BO - RESPONSIVE] Notification pan needs some adjust on mobile
#### Is your feature request related to a problem?

Notification on migrated pages are not as UX as we could

#### Describe the solution you'd like

Scott provided a mockup for this :

image

Actually it looks like this :

image

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Notifications and profile pages are not UX on mobile on legacy pages in the header
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23099.
| How to test?      | Go on a legacy page (like dashboard) and watch notification and profile modals (see screen and issue), be really friendly with this one as it's legacy page, it's pretty hard to do the exact same thing as new-theme one, also clicking on these is a bit "buggy", this is not related to this PR and I can't find how to fix it now :/

image

image

Before :

image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23155)

### CODE REVIEW

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 1]
```diff
@@ -148,6 +161,15 @@ $search-border-color: #bbcdd2;
       padding-top: 11px;
       color: $header-text-color !important;
 
+      @include media-breakpoint-down(sm) {
+        // stylelint-disable-next-line
+        height: 100%!important;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]:
```suggestion
        height: 100% !important;
```

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 2]
```diff
@@ -163,6 +185,11 @@ $search-border-color: #bbcdd2;
       list-style: none;
 
       > a {
+        @include media-breakpoint-down(sm) {
+          // stylelint-disable-next-line
+          height: 100%!important;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]:
```suggestion
          height: 100% !important;
```

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 3]
```diff
@@ -379,6 +434,16 @@ $search-border-color: #bbcdd2;
             a {
               color: #6c868e;
 
+              @include media-breakpoint-down(sm) {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+
+                &, i {
+                  color: #F44336;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]: Could be a variable, it's related to `danger`?

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 4]
```diff
@@ -574,6 +689,27 @@ $search-border-color: #bbcdd2;
     border-radius: 0;
     box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1);
 
+    @include media-breakpoint-down(sm) {
+      left: 0;
+      right: inherit;
+      // stylelint-disable-next-line
+      width: 100vw!important;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]:
```suggestion
      width: 100vw !important;
```

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 5]
```diff
@@ -574,6 +689,27 @@ $search-border-color: #bbcdd2;
...
+      top: 60px;
+      border: none;
+      border-top: 1px solid #EAEBEC;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]:
```suggestion
      border-top: 1px solid $background-grey;
```
Even if I dislike adding color in variable name ^^

File: admin-dev/themes/default/scss/partials/_header.scss [hunk 6]
```diff
@@ -614,6 +767,13 @@ $search-border-color: #bbcdd2;
               color: $brand-info;
               border-top: 0;
               border-bottom: 3px solid $brand-info;
+
+              @include media-breakpoint-down(sm) {
+                &, &:focus, &:active {
+                  background-color: #F4FCFD;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/partials/_header.scss]:
```suggestion
                  background-color: $input-focus-background;
```

File: admin-dev/themes/default/scss/modules/_variables.scss [hunk 7]
```diff
@@ -33,6 +33,7 @@ $gray-dark: lighten(#000, 20%);
 $gray: lighten(#000, 33.5%);
 $gray-light: lighten(#000, 60%);
 $gray-lighter: lighten(#000, 93.5%);
+$gray-background: #eaebec;
```
Comment (PierreRambaud) [inline admin-dev/themes/default/scss/modules/_variables.scss:36]: Duplicate from $background-grey in `_settings.scss` :thinking:

Comment (PierreRambaud) [reply admin-dev/themes/default/scss/modules/_variables.scss:36]: My bad it's in the new-theme

**Review by PierreRambaud — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Comment by SD1982** [comment]
HI @NeOMakinG 
It looks good 
**Before PR**
image
image

**After PR**
image
image


The only "thing" that is not is the notification count that is not well placed (mobile and tablet)
**Before PR**
image
**After PR**
image
image

It's OK on migrated pages
image

**Comment by NeOMakinG** [comment]
@SD1982 nice catch, it should be fixed now

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM now !!
image
image
image
Thanks @NeOMakinG  !!

**Comment by PierreRambaud** [comment]
Thank you @NeOMakinG

**Pull request merged**